EIPS/eip-5749.md

```diff
@@ -0,0 +1,118 @@
+---
+eip: 5749
+title: Deprecate 'window.ethereum'
+description: Deprecate 'window.ethereum' and replace with 'window.evmproviders' to improve UX and spur innovation in the wallet space
+author: Kosala Hemachandra (@kvhnuke), Brett Kolodny (@brettkolodny)
+discussions-to: https://ethereum-magicians.org/t/eip-5749-deprecate-window-ethereum/11195
+status: Draft
+type: Standards Track
+category: Interface
+created: 2022-10-04
+requires: 1193
+---
+
+
+## Abstract
+
+A Javascript Ethereum Provider interface injection for interoperability between multiple wallets at the same time.
+
+## Motivation
+
+At present, `window.ethereum` is the prevailing method by which Ethereum and EVM chain applications interact with injected wallets. This convention originated when adopted by Mist Wallet in 2015 to facilitate a single wallets interaction with applications at a time when no such solution existed. With the proliferation of applications and wallets across multiple EVM chains over the past 7+ years, the adoption of `window.ethereum` as a standard has yielded a number of unintended negative consequences.
+
+- Application UX - The `window.ethereum` standard allows only one wallet to be injected at a time. What results is a race condition between two or more wallets due to all attempting to take over the one `window.ethereum` variable. For users, this creates an inconsistent connection behavior that makes having and using more than one browser wallet unpredictable and impractical. The current solution to this is for for wallets to inject under their own name spaces, but this is not feasible for adoption as every application would need to be made aware of the wallet before the wallet is usable within the application.
+
+- Consolidation of the wallet experience - The race condition created by `window.ethereum` places injected wallets into a 'winner-take-all' relationship that is difficult for users or wallet developers to mitigate. Because of this, users are disincentivized to experiment with new wallets. This creates a 'winner-takes-most' wallet market across EVM chains which forces application developers to optimize for a particular wallet experience.
+
+- Innovation in the wallet space - The 'winner-takes-most' wallet environment that results from the `window.ethereum` standard slows innovation in the wallet space because it creates an artificial barrier to user adoption. New entrants into the space have difficulty gaining traction against legacy players because users are incentivized to have only one injected wallet. With new entrants crowded out under the standard, legacy wallet providers are put under little pressure to innovate.
+
+- Ecosystem UX - Wallets continue to be the most fundamental tool for interacting with blockchains. A homogeneous wallet experience in Ethereum and EVM chains risks stunting UX improvement across the ecosystem and will allow other ecosystems that are more encouraging of competition and innovation to move ahead. 
```

Comment:
```suggestion
- The 'winner-takes-most' wallet environment that results from the `window.ethereum` standard hinders innovation because it creates a barrier to adoption. New entrants into the space have difficulty gaining traction against legacy players because users can have no more than one injected wallet. With new entrants crowded out, legacy wallet providers are put under little pressure to innovate.
- Wallets continue to be the most fundamental tool for interacting with blockchains. A homogeneous wallet experience in Ethereum and EVM chains risks stunting UX improvement across the ecosystem and will allow other ecosystems that are more encouraging of competition and innovation to move ahead. 
```